src/main/java/org/jboss/msc/service/ServiceControllerImpl.java

```diff
@@ -571,12 +570,12 @@ private void transition(final ArrayList<Runnable> tasks) {
                         dependenciesDemanded = false;
                     }
                     getListenerTasks(transition, tasks);
-                    tasks.add(new DependencyStoppedTask(getDependents()));
```

Comment:
There is really no functional change @dmlloyd :)
The controller dependents are initialized in DependentsControllerTask constructor
and all tasks are always instantiated under intrinsic lock. See previous commit.